### ISSUE TITLE
Libre Office top bar unreadable

### ISSUE THREAD

**Issue by padeiro443** [tista500/Adapta#131]
**The top bar of libre office is too while including the text, because of that is very harda to read**

 ```
 * Distribution -
 * Gtk+ 3.20 version -
 * Gnome-Shekk - 
 ```

**Adapta version (latest)**

**LibreOffice**

**Actual issue**

**Steps to reproduce (if you knew)**

**Expected behaviour**

**Other Note (feature-request, question, etc...)**

**Comment by GhettoGirl** [comment]
#94, #35, #43 
> tista500 wrote:
> And this is not theme-related, but happens with the combo "Gtk+ 3.20 + LibreOffice 5.1.1" + dark menubar on light variant". Seems LibreOffice could not run on Gtk+ 3.20.

> Use Gtk+ 2.0 backend.

This is a bug in LibreOffice itself, and cannot be fixed. Workaround: tell LibreOffice to use Gtk+ 2.0 instead.

```bash
export OOO_FORCE_DESKTOP=gtk2
```

Regards

**Comment by gregorydk** [comment]
This bug (https://github.com/tista500/Adapta/issues/43) seems to be solved in LibreOffice 5.2 beta.

**Comment by tista500** [comment]
Greetings,

Fix was committed: 
https://github.com/tista500/Adapta/commit/07714caa2c6b260cefdb8cafc9af3b44cd0bbb2d

Regards.
